### PR TITLE
(1)Remove autotools and some unused files.

### PR DESCRIPTION
**Pull request by usamimikamen-sk**
I think autotools should be removed first because maintenance costs are large.

Other unused or obsolete files should be removed.
* ScrollKeeper's OMF files.
* gnucash-docs.spec.in.
* CMakeNotes.txt
* HACKING

### CODE REVIEW

**Comment by jralls** [comment]
Good start, but 3 commits please: 1 each for autotools, OMF files, and other stuff.

**Comment by gjanssens** [comment]
Ah, what a relief to finally see these go. And I agree completely it makes sense to drop autotools before anything else. It saves on refactoring work for your other PRs.

**Comment by gjanssens** [comment]
Thanks!

File: CMakeLists.txt [hunk 1]
```diff
@@ -196,7 +196,6 @@ add_to_dist(
     COPYING
     COPYING-DOCS
     ChangeLog
-    HACKING
```
Comment (fellen) [inline CMakeLists.txt:218]: This file is still mentioned in README.

Tip: Before removing files run a command like `grep -inr 'HACKING'` in the top source directory.

Comment (gjanssens) [reply CMakeLists.txt:218]: Thanks for the heads up @fellen . I assume you will handle this trivial fix yourself ?

**Pull request merged**